src/pip/_internal/req/req_install.py

```diff
@@ -777,18 +777,49 @@ def install(
 
         global_options = global_options if global_options is not None else []
         if self.editable:
```

Comment:
Somehow I think this should become `if self.editable and not self.is_wheel` then legacy install, else go ahead and install wheel with the regular wheel below, no other change necessary in this function.

Reply:
We'll have to decide if this prototype makes sense or if it is just the way pip works. The prototyped approach is to put extra files into the directory into which we have already generated our `*.dist-info` directory as part of another hook. It's not quite a wheel. It's not necessarily easy for the build system, accustomed to building a regular wheel, to build a completely different kind of wheel. Stub modules in place of the regular contents, no need for compatibility tags, wheel caching, *.data directories, or the RECORD associated with a full wheel.